lib/Github/Api/CurrentUser.php

```diff
@@ -178,4 +178,15 @@ public function installations(array $params = array())
     {
         return $this->get('/user/installations', array_merge(array('page' => 1), $params));
     }
+
+    /**
+     * @link https://developer.github.com/v3/integrations/installations/#list-repositories-accessible-to-the-user-for-an-installation
+     *
+     * @param string $installationId  the ID of the Installation
+     * @param array $params
+     */
+    public function repositoriesByInstallation($installationId, array $params = array())
+    {
+        return $this->get("/user/installations/$installationId/repositories", array_merge(array('page' => 1), $params));
```

Comment:
Page is only hardcoded when you don't pass a page array key in the params array. So this is the default value